src/region/RegionManager.tsx

```diff
@@ -36,7 +36,7 @@ export default class RegionManager implements BaseManager {
         rootLayerEl.setAttribute('data-resin-feature', 'annotations');
 
         // Insert the new layer element immediately after the reference element
-        return parentEl.insertBefore(rootLayerEl, referenceEl.nextSibling);
+        return parentEl.insertBefore(rootLayerEl, referenceEl);
```

Comment:
We only want it to go under the `.textLayer` which may not always be present on a document -- so we may need to add some logic around that

Reply:
Managers should not be modified to work for specific viewers. Does this work for image annotations? We need to be very mindful about changing existing features.